www/js/userInfoService.js

```diff
@@ -0,0 +1,57 @@
+var OFFLINE_ONLY = true;
```

Comment:
Yah, I'm just following your examples on this... but a demo/production configuration would definitely work for the future